src/contracts/proxies/NFTRenderer.sol

```diff
@@ -281,10 +283,15 @@ contract NFTRenderer {
   /**
    * @dev calculates liquidation risk
    */
-  function _calcRisk(uint256 ratio) internal pure returns (string memory, string memory) {
-    if (ratio < 120) return ('LIQUIDATION', '#E45200');
-    else if (ratio > 119 && ratio < 136) return ('HIGH', '#E45200');
-    else if (ratio > 135 && ratio < 150) return ('ELEVATED', '#FCBF3B');
+  function _calcRisk(
+    uint256 ratio,
+    uint256 liquidationRatio,
+    uint256 safetyRatio
+  ) internal pure returns (string memory, string memory) {
+    if (ratio == 0) return ('LOW', '#459d00');
```

Comment:
We added a closed state when the vault is empty. Is the risk "low" in this case. As long as it matches the app.

Reply:
I saw this PR so I wanted to mention that we added this scenario:
```js
if (currentLiquidationRatioAsDecimal === liqRatio) return 3 (aka elevated)
```
and 
```js
if (currentLiquidationRatioAsDecimal < liqRatio) return 4 (liquidation)
```

so I think the solidity should be changed to 

```solidity
if (ratio < liquidationRatio) return ('LIQUIDATION', '#E45200');
```
and add the case

```solidity
if (ratio = liquidationRatio) return ('ELEVATED', '#FCBF3B');
```